cluster-scope/components/centos/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
The `cluster-scope` folder is for resources that project admins are not allowed to deploy.

The folders `centos` and `kvm-device-plugin` largely contain `namespace` scoped resources that a project admin can deploy, so they should be moved to the root of the `apps` repo.

`Namespace` resources are cluster scoped, so they should be included in the `cluster-scope/base/core/namespaces` folder. 

The `daemonset` requires privileged access, so that too should be in `cluster-scope` (admittedly cluster-scope is misleading here, as `daemonset` is namespace scoped). 

The rest can be moved to the root of the apps repo, please follow the following structure: 


```
.
└── apps
    ├── centos
    │   ├── base
    │   │   └── kustomization.yaml
    │   └── overlays
    │       └── smaug
    │           └── kustomization.yaml
    ├── cluster-scope
    │   └── base
    │       ├── apps
    │       │   └── daemonsets
    │       │       └── kvm-device-plugin
    │       │           └── kustomization.yaml
    │       └── core
    │           └── namespaces
    │               ├── centos
    │               └── kvm-device-plugin
    └── kvm-device-plugin
        ├── base
        │   └── kustomization.yaml
        └── overlays
            └── smaug
                └── kustomization.yaml
```

Include any `cluster-scope/base/*` resources that need to be added to the `smaug` cluster [here](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/moc/smaug/kustomization.yaml).

Reply:
I've renamed the ns to `okd-centos` to clarify that it is maintained by the OKD team